pkgs/tools/misc/wayback/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, pkg-config
+, libtool
+, google-chrome
```

```suggestion
, chromium
```
They recommend chromium on the official website: https://docs.wabarc.eu.org/installation/
Also youtube-dl could be useful too